example.config.env

```diff
@@ -11,7 +11,7 @@ NODE_ENV="development"
 
 SESS_NAME="sid"
 SESS_SECRET="something-super-duper-secret"
-SESS_LIFE=3600000
+SESS_LIFE=36000000
```

Comment:
please revert this back to the original value, I don't think we want the session to last for 600 hours